### PR TITLE
[master < T1153] Disk storage improvements 

### PR DESCRIPTION
**Pull request by as51340**
[master < Epic] PR
- [ ] Check, and update documentation if necessary
- [ ] Write E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch and the Epic branch
- [ ] Provide the full content or a guide for the final git message

[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [ ] Write a release note here, including added/changed clauses
- [ ] Tag someone from docs team in the comments

### CODE REVIEW

**Review by Darych — APPROVED** [review]

File: src/storage/v2/disk/storage.cpp [hunk 1]
```diff
@@ -106,7 +106,9 @@ bool VertexHasLabel(const Vertex &vertex, LabelId label, Transaction *transactio
         break;
       }
       case Delta::Action::DELETE_DESERIALIZED_OBJECT:
-      case Delta::Action::DELETE_OBJECT:
+      case Delta::Action::DELETE_OBJECT: {
+        break;
+      }
```
Comment (andrejtonev) [inline src/storage/v2/disk/storage.cpp:111]: Same error at line 140

**Review by andrejtonev — APPROVED** [review]

File: src/storage/v2/disk/storage.cpp (hunk 1 above)
Comment (as51340) [reply src/storage/v2/disk/storage.cpp:111]: can you explain?

Comment (andrejtonev) [reply src/storage/v2/disk/storage.cpp:111]: `break` was missing after `DELETE_OBJECT`
There is a similar switch at line 140 where the break is missing as well.

Comment (as51340) [reply src/storage/v2/disk/storage.cpp:111]: was missing but now is okay or? Sorry I don't understand :smile: And about line 140, I think there is missing break yes I will add it now

Comment (andrejtonev) [reply src/storage/v2/disk/storage.cpp:111]: this change is correct :D yes

**Review by Darych — APPROVED** [review]

**Pull request merged**